android/assets/html-es/about2d.html

```diff
@@ -6,7 +6,7 @@
 <link href="../style.css" rel="stylesheet" type="text/css"/>
 </head>
 <body>
-<p><strong> Barcode Scanner </strong> También entiende cómo leer dos códigos de barras bidimensionales,
+<p><strong> Barcode Scanner </strong> También entiende cómo leer los códigos de barras bidimensionales,
```

Comment:
Thanks for taking the time to do this :) 
Not that I can approve this, but I thought I'd help out.
I understand that "también" should not be capitalized, because it is not at the beginning of the phrase.